lib/order.rb

```diff
@@ -0,0 +1,36 @@
+require './lib/takeaway'
+require './lib/message'
+require 'twilio-ruby'
+
+class Order
+
+  attr_reader :order
```

Comment:
It's a _**bit**_ confusing to have an attribute called `order` on an instance of `Order`. Makes me think of [this](https://www.youtube.com/watch?v=H4v7wddN-Wg).